src/smc-util/message.js

```diff
@@ -1189,13 +1191,14 @@ API(
       }
     },
     desc: `\
-Create a text file in the target project.
-User must be owner or collaborator in the target project.
+Create a text file in the target project with the given \`project_id\`.
 Directories containing the file are created if they do not exist already.
-Unix user in the target project must have permissions to create file
-and containing directories if they do not already exist.
 If a file already exists at the destination path, it is overwritten.
 
+**Note:** You need to have read access to the project.
+The Linux user \`user\` in the target project must have permissions to create files
```

Comment:
This sentence only makes sense in the context of Kubernetes. It doesn't make sense for other ways of running Cocalc, e.g., in the docker image.   I think I'll merge this anyways, but be aware that in Docker-cocalc, there is no `user` user.